### ISSUE TITLE
Remove "see all" links

### ISSUE THREAD

**Issue by NomeQ** [Trillium-Solutions/mendocino-wp-theme#9]
Links with text like "click here" and "see all" are inaccessible to screen readers. Remove "see all" links from homepage and anywhere else they occur, and instead just make the relevant text (e.g. "News" or "Alerts") into a link.

**Comment by NomeQ** [comment]
See all links are still present
![Screen Shot 2021-02-04 at 5 56 31 PM](https://user-images.githubusercontent.com/9342771/106979092-70888700-6712-11eb-80b1-95c373b614a7.png)

In this case, the title, "Service Alerts" should be a link to the see all page, and there should be hover/focus styles to ensure that the user knows it is a link.

**Comment by NomeQ** [comment]
Wowowow yep didn't hard refresh.
